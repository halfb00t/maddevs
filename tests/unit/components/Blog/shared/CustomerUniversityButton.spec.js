import { shallowMount } from '@vue/test-utils'
import CustomerUniversityButton from '@/components/Blog/shared/CustomerUniversityButton'

const props = {
  label: 'Click me',
  size: 'md',
}

describe('CustomerUniversityButton component', () => {
  it('should correctly render', () => {
    const wrapper = shallowMount(CustomerUniversityButton, {
      propsData: props,
    })

    wrapper.find('.customer-university__btn').trigger('click')
    expect(wrapper.emitted().click).toBeTruthy()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
