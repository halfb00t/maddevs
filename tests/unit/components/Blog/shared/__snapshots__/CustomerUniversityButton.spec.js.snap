// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomerUniversityButton component should correctly render 1`] = `
<button class="customer-university__btn customer-university__btn--md">
  Click me
</button>
`;
